python-lib/sharepoint_client.py

```diff
@@ -274,7 +274,7 @@ def is_response_empty(response):
     def extract_results(response):
         return response[SharePointConstants.RESULTS_CONTAINER_V2][SharePointConstants.RESULTS]
 
-    def get_list_items(self, list_title, query_string=""):
+    def get_list_items(self, list_title, params={}):
```

Comment:
avoid mutable object as default value, `params=None` is safer

Reply:
👍